app/views/fact_values/_fact.html.erb

```diff
@@ -8,14 +8,13 @@
         <% end %>
       </td>
   <% end %>
-  <td>
+  <td class="ellipsis">
     <%= fact_name(fact, parent) %>
   </td>
   <td class="fact-col">
     <% unless fact.compose %>
       <%= link_to trunc_with_tooltip(fact.value, 40),
                   fact_values_path("search" => "facts.#{fact.name} = \"#{fact.value}\""),
-                  :title => _("Show all %{name} facts where they are equal to %{value}") % { :name => fact.name, :value => fact.value },
```

Comment:
while not entirely related to the issue, this was displaying two conflicting tooltips, there's not much value in this text anyway so I removed it